### PR TITLE
Add comments and group JPEGd treasury addresses

### PR DESCRIPTION
**Pull request by 0xmichalis**

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/treasury/jpegd.js exports TVL: 
        
 
 
    ethereum-ownTokens        13.82 M
    ethereum                  52.17 M
    ownTokens                 13.82 M
    
    total                    52.17 M

File: projects/treasury/jpegd.js [hunk 1]
```diff
@@ -25,17 +25,18 @@ const treasuryTvl = treasuryExports({
...
       ADDRESSES.ethereum.RETH,
-      lp,
-      lp2,
+      "0x836A808d4828586A69364065A1e064609F5078c7", // pETH
+      // Liquidity positions
+      "0xC47EBd6c0f68fD5963005D28D0ba533750E5C11B", // pUSDFRAXBP3CRV-f
+      "0x34eD182D0812D119c92907852D2B429f095A9b07", // JPEGETH-f
+      "0x9848482da3Ee3076165ce6497eDA906E66bB85C5", // pETH-ETH-f
+      "0xdB06a76733528761Eda47d356647297bC35a98BD", // JPEGWETH SLP
+      "0xC36442b4a4522E871399CD717aBDD847Ab11FE88", // UniV3 positions
```
Comment (g1nt0ki) [inline projects/treasury/jpegd.js:39]: you can skip this as it is already included thanks to this flag `resolveUniV3: true,`

Comment (0xmichalis) [reply projects/treasury/jpegd.js:39]: removed

**Comment by llamatester** [comment]
The adapter at projects/treasury/jpegd.js exports TVL: 
        
 
 
    ethereum-ownTokens        13.92 M
    ethereum                  52.21 M
    ownTokens                 13.92 M
    
    total                    52.21 M

**Pull request merged**